src/r_bsp.c

```diff
@@ -284,11 +284,11 @@ static void R_MaybeInterpolateSector(sector_t* sector)
         sector->oldgametic == gametic - 1)
     {
         // Interpolate between current and last floor/ceiling position.
-        if (sector->floorheight != sector->oldfloorheight)
+        if (sector->floordata && sector->floorheight != sector->oldfloorheight)
```

Comment:
Shouldn't `sector->floorheight == sector->oldfloorheight` for sectors without floor thinkers anyway? Same for ceiling.

Reply:
Apparently not always, this map has an interesting effect. I took the code from PrBoom+:
https://github.com/coelckers/prboom-plus/blob/3e957d68a853375a0cf8c3908a0c745206031949/prboom2/src/r_fps.c#L512-L515

Reply:
Maybe if a sector's floor or ceiling was once miving, but then the movement stopped and the "oldheight" value isn't updated anymore. Anyway, thank you once again!